tools/clang_format.py

```diff
@@ -42,27 +50,15 @@
 #
 
 # Expected version of clang-format
-CLANG_FORMAT_VERSION = "3.8.0"
-CLANG_FORMAT_SHORT_VERSION = "3.8"
+CLANG_FORMAT_VERSION = "5.0.1"
+CLANG_FORMAT_SHORT_VERSION = "5.0"
```

Comment:
So, this is OK because you know we're running this only on machines with cool distros, right?

Reply:
I need to figure out which version of clang-format we want to dictate, so formatting will be the same for all contributors.

Reply:
well, that's then going to be a problem – different packagers package different versions. I wonder if we can have something like a "reformatting" push remote, which lets you push, then wait a couple second, then lets you pull, giving you a `--fixup` commit with the reformatted / `clang-tidy`ied files.

Reply:
Across all major distributions you have access to at least two or more compiler major versions.

Reply:
I's 4.0.1 now, which I believe we can expect people to have available.

Reply:
Agreed. FYI, we did the same for UHD.